// @vkontakte/vkjs/lib/other/index.js
/**
 * classNames
 */ export { clsx as classNames } from "clsx";
/**
 * clipboard
 */ export { copyTextToClipboard } from "./clipboard.js";
/**
 * common
 */ export { isNumeric } from "./common.js";
/**
 * detections
 */ export { isPassiveEventsSupported, isSmoothScrollSupported } from "./detections.js";
export { isEqual } from "./equal.js";
export { noop, once } from "./functions.js";
export { getCookie, isCookieEnabled } from "./cookie.js";
/**
 * OffsetRect
 */ export { getOffsetRect } from "./getOffsetRect.js";
export { getPhotoSize } from "./getPhotoSize.js";
/**
 * objects
 */ export { deleteObjectKeys } from "./objects.js";
/**
 * querystring
 */ export { querystring } from "./querystring.js";
/**
 * random
 */ export { getRandomInt, getRandomString } from "./random.js";
export { hasReactNode, isPrimitiveReactNode } from "./react_utils.js";
export { escapeRegExp } from "./regexp.js";
export { localStorage, sessionStorage } from "./storage.js";
export { canUseDOM, canUseEventListeners, onDOMLoaded } from "./dom.js";
